apps/graphql/src/apps/itinerary/helpers/Itineraries.js

```diff
@@ -37,10 +38,19 @@ export const mapVehicle = (type: ?string, uniqueNo: ?string) => ({
   uniqueNo: uniqueNo ?? null,
 });
 
-export const sanitizeCarrier = (routeItem: ?ApiRouteItem) => ({
-  name: null, // @TODO - value is currently missing from API endpoint response
-  code: routeItem?.airline ?? null,
-});
+export const sanitizeCarrier = (routeItem: ?ApiRouteItem) => {
+  // @TODO - value is currently missing from API endpoint response
+  // JSON file is used from https://github.com/BesrourMS/Airlines/blob/master/airlines.json
+  const iata = routeItem?.airline;
+  let name = '';
+  for (let index = 0; index < airlines.length; index++) {
```

Comment:
You can probably do something here like (accounting for null values ofc):

```js
name = airlines.filter(airline => airline['iata'] == iata)[0].name
```

Reply:
I think even `airlines.find(...)` since find will terminate on the first occurrence, and there should be only one. filter will loop through the entire array.

Reply:
Updated to `airlines.find(...)`